src/main/java/service/impl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package service.impl;

```

Comment:
The package declaration should not begin with an empty line. Please remove the empty line above the package declaration to follow Java conventions.